mobile/assets/i18n/de-DE.json

```diff
@@ -208,6 +208,7 @@
   "image_viewer_page_state_provider_download_error": "Fehler beim Herunterladen",
   "image_viewer_page_state_provider_download_success": "Erfolgreich heruntergeladen",
   "image_viewer_page_state_provider_share_error": "Fehler beim Teilen",
+  "jump_to_date": "Zu Datum springen",
```

Comment:
We don't need the keys in language files other than en-US. The other translations are handled through Localizely.